### PR TITLE
Part 1 - Declarations

### PR DESCRIPTION
**Pull request by transistasis**
In this pull request:
* Completed all declarations
* Verified that there are no compiler errors

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -61,9 +75,32 @@ void variableDeclarations()
...
+    bool enablePotatoGraphics = true;
+
+    auto importantMessage = "This is a string.";
+    auto transconductance = 50E-3;
+    auto reverseSaturationCurrent = 10E-15;
 
-    
-    ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+    ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+    ignoreUnused(channelNumber, numberOfTaps, fftPoints, compressionRatio, roomSize, decayValue, attackValue, releaseValue, sustainValue, c, a, t, pluginEnabled, displaySpectrumAnalyzer, enablePotatoGraphics, importantMessage, transconductance, reverseSaturationCurrent);
```
Comment (matkatmusic) [inline main.cpp:103]: don't call `ignoreUnused()` twice. 
call it once and pass all of the unused variables to it.
- [ ] revise

File: main.cpp [hunk 2]
```diff
@@ -137,27 +233,40 @@ int main()
...
+    auto resistanceCalculation = computeResistance(5.0, 100.0E-3);
+
     //10)
-    
-    
-    ignoreUnused(carRented);
+    auto powerCalculation = computePower(50.0E-6f, 1000.0);
+
+    ignoreUnused(carRented);
+    ignoreUnused(integerSum, charIntegerValue, hypotenuseLength, integersAreEqual, arithmeticLogicUnitSum, dotProductResult, forceCalculation, resistanceCalculation, powerCalculation);
```
Comment (matkatmusic) [inline main.cpp:269]: don't call `ignoreUnused()` twice. 
call it once and pass all of the unused variables to it.
- [ ] revise

**Review by matkatmusic — CHANGES_REQUESTED** [review]

File: main.cpp (hunk 1 above)
Comment (transistasis) [reply main.cpp]: Got it! I eliminated the repetition in the `ignoreUnused()` function calls.

File: main.cpp (hunk 2 above)
Comment (transistasis) [reply main.cpp]: I fixed this `ignoreUnused()` function call to adhere to the D.R.Y. principle as well.

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**